compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/ConstantValueResolver.java

```diff
@@ -311,6 +311,8 @@ private BLangConstantValue calculateConstValue(BLangConstantValue lhs, BLangCons
                     return calculateMultiplication(lhs, rhs);
                 case DIV:
                     return calculateDivision(lhs, rhs);
+                case MOD:
```

Comment:
Shall we add more test cases to cover the spec related to remainder operator (to cover edge cases mentioned in the spec) https://ballerina.io/spec/lang/master/#section_6.23

Reply:
Sure.